docs/attributes.md

```diff
@@ -215,6 +215,26 @@ A symbol is functional if it is given the `functional` attribute or the
 
 ## misc.
 
+### Location
+
+```
+org'Stop'kframework'Stop'attributes'Stop'Location{}("Location(sl,sc,el,ec)")
```

Comment:
Are any of `sl`, `sc`, `el`, or `ec` optional?

Reply:
No, it's all-or-nothing from the front-end.